addons/l10n_it_edi/models/account_edi_format.py

```diff
@@ -748,6 +761,12 @@ def _import_fattura_pa_line(self, element, invoice_line_form, extra_info=None):
                     self.env['account.move']._compose_info_message(element, '.'),
                 ))
 
+        # If no taxes were found, try to find fitting taxes
+        if not invoice_line_form.tax_ids:
+            fitting_taxes = invoice_line_form._predict_taxes()
+            if fitting_taxes:
+                invoice_line_form.tax_ids += fitting_taxes
```

Comment:
Probably being picky here but the `+=` doesn't really make sense with the purpose of the commit, does it?

```suggestion
                invoice_line_form.tax_ids = fitting_taxes
```